Unable to upgrade kr

```
==> Upgrading 1 outdated package:
kryptco/tap/kr HEAD-309230e -> HEAD-309230e
==> Upgrading kryptco/tap/kr
==> Cloning https://github.com/kryptco/kr.git
Cloning into '/Users/Henry/Library/Caches/Homebrew/kr--git'...
==> Checking out branch master
Already on 'master'
Your branch is up to date with 'origin/master'.
Submodule 'sigchain' (https://github.com/kryptco/sigchain) registered for path 'sigchain'
Cloning into '/Users/Henry/Library/Caches/Homebrew/kr--git/sigchain'...
Submodule path 'sigchain': checked out '8ad8442c55a3363d44340fb07c018fa4d9362f2b'
Submodule 'libsodium' (https://github.com/jedisct1/libsodium) registered for path 'sigchain/libsodium'
Submodule 'openssl' (https://github.com/openssl/openssl) registered for path 'sigchain/openssl'
Cloning into '/Users/Henry/Library/Caches/Homebrew/kr--git/sigchain/libsodium'...
Cloning into '/Users/Henry/Library/Caches/Homebrew/kr--git/sigchain/openssl'...
Submodule path 'sigchain/libsodium': checked out '5ebc0d40295ac6c509aaf1d3ed5ded6bc2a77074'
Submodule path 'sigchain/openssl': checked out 'd4e4bd2a8163f355fa8a3884077eaec7adc75ff7'
/Users/Henry/Library/Caches/Homebrew/kr--git/sigchain
/Users/Henry/Library/Caches/Homebrew/kr--git/sigchain/libsodium
/Users/Henry/Library/Caches/Homebrew/kr--git/sigchain/openssl
Warning: Calling MacOS.prefer_64_bit? is deprecated! There is no replacement.
Please report this to the kryptco/tap tap:
  /usr/local/Homebrew/Library/Taps/kryptco/homebrew-tap/kr.rb:38

==> mkdir -p /private/tmp/kr-20190124-61026-ak90jo/.brew_home
==> emcc
==> sed -i -e s/^BINARYEN_ROOT.*/BINARYEN_ROOT = \'\/usr\/local\/opt\/binaryen\'
==> sed -i  /^LLVM_ROOT/d /private/tmp/kr-20190124-61026-ak90jo/.brew_home/.emsc
==> sh -c echo LLVM_ROOT = \'/usr/local/opt/emscripten/libexec/llvm/bin\' >> /pr
==> sed -i -e s/^NODE_JS.*/NODE_JS = \'\/usr\/local\/bin\/node\'/ /private/tmp/k
==> rustup-init -y
==> rustup target add wasm32-unknown-emscripten
==> cargo install --debug --version=0.6.10 cargo-web
==> make install
Last 15 lines from /Users/Henry/Library/Logs/Homebrew/kr/10.make:

rm -rf /private/tmp/kr-20190124-61026-ak90jo/src/github.com/kryptco/kr/Frameworks/krbtle.framework
cp -R krbtle/build/Release/krbtle.framework /private/tmp/kr-20190124-61026-ak90jo/src/github.com/kryptco/kr/Frameworks/krbtle.framework
cd sigchain && CARGO_RELEASE=--release make libsigchain-with-dashboard
cargo web --version || (echo 'Please install cargo-web with `cargo install cargo-web`and make sure $HOME/.cargo/bin is on your $PATH' ; exit 1;)
cargo-web 0.6.10
# use rsync so that file modifed times are only updated when the contents change
cd dashboard_yew && cargo web deploy --release --target=wasm32-unknown-emscripten && rsync --checksum --delete -r ../target/deploy/* ../target/deploy-final
error: error: failed to parse manifest at `/private/tmp/kr-20190124-61026-ak90jo/src/github.com/kryptco/kr/sigchain/Cargo.toml`

Caused by:
  virtual manifests do not specify [target]

make[1]: *** [libsigchain-with-dashboard] Error 101
make: *** [all] Error 2
```


OS: macOS 10.14.2

--- a/kr.rb
+++ b/kr.rb
@@ -35,7 +35,7 @@ class Kr < Formula
   def install
     ENV["GOPATH"] = buildpath
     ENV["GOOS"] = "darwin"
-    ENV["GOARCH"] = MacOS.prefer_64_bit? ? "amd64" : "386"
+    ENV["GOARCH"] = Hardware::CPU.is_64_bit? ? "amd64" : "386"
 
     dir = buildpath/"src/github.com/kryptco/kr"
     dir.install buildpath.children

